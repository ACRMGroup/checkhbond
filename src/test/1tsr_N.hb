residue ARG
donate	      30	      30	      30	    17
partnertodonate	      26	      27	      30	     1
