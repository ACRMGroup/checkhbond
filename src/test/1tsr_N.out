ARG    202 N    : ASN    200 OD1  H    CG  
ASN    263 N    : SER    261 OG   H    CB  
ASP    184 N    : SER    183 OG   H    CB  
ASP    186 N    : SER    185 OG   H    CB  
SER    260 N    : ASP    259 OD1  H    CG  
HIS    168 N    : GLN    167 OE1  H    CD  
HIS    168 N    : GLN    167 OE1  H    CD  
ALA    189 N    : ASP    186 OD1  H    CG  
GLY    187 N    : SER    185 OG   H    CB  
LEU    188 N    : ASP    186 OD1  H    CG  
VAL    197 N    : TYR    205 OH   H    CZ  
VAL    274 N    : SER    240 OG   H    CB  
